### PR TITLE
update(HTML): web/html/element/input/time

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [&lt;input type="time"&gt;@MDN](https://developer.mozilla.org/en-us/docs/Web/HTML/Element/input/time), [сирці &lt;input type="time"&gt;@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/html/element/input/time/index.md)

Нові зміни:
- [Convert noteblocks for web/html/element folder (#35085)](https://github.com/mdn/content/commit/b7955e77cd4293adf45ef23686df50b0305f02ad)
- [Fix code-link containing spaces, part 1 (#35065)](https://github.com/mdn/content/commit/998a4e6fc713678381e6bc31130e3a0a53158d69)
- [enhance: html input element - technical summary - attributes (#31639)](https://github.com/mdn/content/commit/68ee1b1496bfd5dd779feed417e52a91fa55d789)
- [Fix anchors, JS/HTTP/HTML (#34539)](https://github.com/mdn/content/commit/d7e274e727920f0f85f14e0bdd18e6e585419a90)
- [Update link to date and time picker (#34134)](https://github.com/mdn/content/commit/222d80ecb8d927becc8e8999e9edbbc3b9000ddd)
- [time input: remove fallbacks (#33924)](https://github.com/mdn/content/commit/cd02f879d33689b5769d545fb6851bf055686dec)

### CODE REVIEW

File: files/uk/web/html/element/input/time/index.md [hunk 1]
```diff
@@ -159,9 +129,7 @@ _Наразі неясно, що означає значення `any` для `s
 
 Для зміни кількості часу, на котру відбувається стрибок при інкременті або декременті, можна скористатися атрибутом [`step`](/uk/docs/Web/HTML/Element/input#step-krok) (наприклад, так, щоб час змінювався на 10 хвилин при кожному клацанні маленьких віджетів стрілок).
 
-> **Примітка:** Ця властивість має деякі дивні ефекти в різних браузерах, тому не є повністю надійною.
-
-Цей атрибут приймає ціле числове значення, рівне числу секунд, на котре повинен відбуватися інкремент; усталене значення – 60 секунд, тобто одна хвилина. Якщо задати значення, менше за 60 секунд (1 хвилину), то поле `time` покаже віконце для введення секунд поряд з годинами та хвилинами:
+Цей атрибут приймає ціле числове значення, що визначає кількість секунд, на які повинен відбувати інкремент; усталене значення – 60 секунд. Коли вживається це усталене значення, більшість UI часу користувацьких агентів виводять години та хвилини, але не секунди. Додання атрибуту [`step`](/uk/docs/Web/HTML/Element/input#step-krok) з будь-яким числовим значенням, що не ділиться на `60`, додає до UI секунди, якщо до появи секунд не призвели значення `min` або `max`.
```
[the remark this answers is not in the document]
Comment (undead404) [reply files/uk/web/html/element/input/time/index.md:132]:
```suggestion
Цей атрибут приймає ціле числове значення, що визначає кількість секунд, на які повинен відбувати інкремент; усталене значення – 60 секунд. Коли вживається це усталене значення, більшість UI часу користувацьких агентів виводять години та хвилини, але не секунди. Додання атрибута [`step`](/uk/docs/Web/HTML/Element/input#step-krok) з будь-яким числовим значенням, що не ділиться на `60`, додає до UI секунди, якщо до появи секунд не призвели значення `min` або `max`.
```

File: files/uk/web/html/element/input/time/index.md [hunk 2]
```diff
@@ -9,40 +9,10 @@ browser-compat: html.elements.input.type_time
 
 Елементи {{HTMLElement("input")}} типу **`time`** (час) утворюють поля введення, призначені для того, щоб дозволити користувачам легко вводити час (години та хвилини, а також, за бажанням, секунди).
 
-Користувацький інтерфейс цього елемента керування відрізняється в різних браузерах; дивіться докладніше про це в розділі [Сумісність із браузерами](#sumisnist-iz-brauzeramy). У браузерах, що його не підтримують, цей елемент доладно відступає до [`<input type="text">`](/uk/docs/Web/HTML/Element/input/text).
+Попри те, що зовнішній вигляд користувацького інтерфейсу цього контрольного елемента залежить від браузера та операційної системи, можливості в нього одні й ті ж. Значення завжди є 24-годинним відформатованим часом `hh:mm` або `hh:mm:ss`, з додатковими нулями, незалежно від формату введення UI.
```
Comment (alinkedd) [inline files/uk/web/html/element/input/time/index.md:12]:
```suggestion
Попри те, що зовнішній вигляд користувацького інтерфейсу цього контрольного елемента залежить від браузера та операційної системи, можливості в нього одні й ті ж. Значення завжди є 24-годинним відформатованим часом `hh:mm` або `hh:mm:ss` з провідними нулями, незалежно від формату введення UI.
```
https://uk.wikipedia.org/wiki/%D0%9F%D1%80%D0%BE%D0%B2%D1%96%D0%B4%D0%BD%D1%96_%D0%BD%D1%83%D0%BB%D1%96

File: files/uk/web/html/element/input/time/index.md [hunk 3]
```diff
@@ -134,7 +104,7 @@ startTime.addEventListener(
 
 Для полів `time` значення `step` дається в секундах, з масштабним фактором 1000 (оскільки за лаштунками числове значення зберігається в мілісекундах). Усталене значення `step` – 60, що відповідає 60 секундам (або 1 хвилині, або 60.000 мілісекунд).
 
-_Наразі неясно, що означає значення `any` для `step`, коли воно використовується з полями `time`. Ця інформація буде оновлена, щойно буде визначена._
+Коли як значення `step` задано `any`, то вживається усталене значення 60 секунд, а секундне значення не виводиться в UI.
```
Comment (alinkedd) [inline files/uk/web/html/element/input/time/index.md:107]:
```suggestion
Коли `step` має значення `any`, то вживається усталене значення 60 секунд, а секундне значення не виводиться в UI.
```

File: files/uk/web/html/element/input/time/index.md [hunk 4]
```diff
@@ -159,9 +129,7 @@ _Наразі неясно, що означає значення `any` для `s
 
 Для зміни кількості часу, на котру відбувається стрибок при інкременті або декременті, можна скористатися атрибутом [`step`](/uk/docs/Web/HTML/Element/input#step-krok) (наприклад, так, щоб час змінювався на 10 хвилин при кожному клацанні маленьких віджетів стрілок).
 
-> **Примітка:** Ця властивість має деякі дивні ефекти в різних браузерах, тому не є повністю надійною.
-
-Цей атрибут приймає ціле числове значення, рівне числу секунд, на котре повинен відбуватися інкремент; усталене значення – 60 секунд, тобто одна хвилина. Якщо задати значення, менше за 60 секунд (1 хвилину), то поле `time` покаже віконце для введення секунд поряд з годинами та хвилинами:
+Цей атрибут приймає ціле числове значення, що визначає кількість секунд, на які повинен відбувати інкремент; усталене значення – 60 секунд. Коли вживається це усталене значення, більшість UI часу користувацьких агентів виводять години та хвилини, але не секунди. Додання атрибута [`step`](/uk/docs/Web/HTML/Element/input#step-krok) з будь-яким числовим значенням, що не ділиться на `60`, додає до UI секунди, якщо до появи секунд не призвели значення `min` або `max`.
```
Comment (alinkedd) [inline files/uk/web/html/element/input/time/index.md:132]:
```suggestion
Цей атрибут приймає ціле числове значення, що визначає кількість секунд, на яку відбувається інкремент; усталене значення – 60 секунд. Коли вживається це усталене значення, більшість UI часу в користувацьких агентах виводить години та хвилини, але не секунди. Додання атрибута [`step`](/uk/docs/Web/HTML/Element/input#step-krok) з будь-яким числовим значенням, що не ділиться на `60`, додає до UI секунди, якщо цього ще не зробили значення `min` або `max`.
```

File: files/uk/web/html/element/input/time/index.md [hunk 5]
```diff
@@ -172,17 +140,11 @@ _Наразі неясно, що означає значення `any` для `s
...
 {{EmbedLiveSample('vykorystannia-atrybuta-step', 600, 40)}}
 
-У Chrome та Opera, котрі є єдиними браузерами, що показують стрілки інкременту та декременту, клацання по стрілках змінює значення секунд на 2 секунди, але не впливає на години або хвилини. Хвилини (або години) можуть використовуватися для стрибків, якщо вказати кількість хвилин (або годин) у секундах, наприклад, 120 для 2 хвилин, або 7200 для 2 годин).
-
-У Firefox стрілок немає, тож значення `step` не застосовується. Проте його присутність _додає_ віконце введення секунд поряд з віконцем введення хвилин.
-
-Схоже на те, що значення кроків не діє в Edge.
-
-> **Примітка:** Схоже, використання `step` призводить до того, що валідація не працює належним чином (як показано в наступному розділі).
+Щоб задати як крок хвилини або години, задайте число хвилин або годин у секундах, наприклад, 120 для 2 хвилин, або 7200 для 2 годин.
```
Comment (alinkedd) [inline files/uk/web/html/element/input/time/index.md:143]:
```suggestion
Щоб задати як крок хвилини або години, задайте число хвилин або годин у секундах, наприклад, 120 для 2 хвилин або 7200 для 2 годин.
```

File: files/uk/web/html/element/input/time/index.md [hunk 6]
```diff
@@ -172,17 +140,11 @@ _Наразі неясно, що означає значення `any` для `s
...
 
 ## Валідація
 
-Усталено `<input type="time">` не застосовує до введених значень жодної валідації, окрім того, що інтерфейс користувацького агента, загалом, не дозволяє ввести нічого, крім значення часу. Це корисно (за умови, що введення часу повністю підтримується користувацьким агентом), але не можна повністю покладатися на те, що значення буде коректним часовим рядком, оскільки воно може бути порожнім рядком (`""`), що допускається. Також можливо, що значення буде приблизно мати вигляд дійсного часу, але не буде коректним, наприклад, `25:05`.
+Усталено `<input type="time">` не застосовує до введених значень жодної валідації, окрім того, що інтерфейс користувацького агента, загалом, не дозволяє ввести нічого, крім значення часу. Це корисно, але не можна повністю покладатися на те, що значення буде коректним часовим рядком, оскільки воно може бути порожнім рядком (`""`), що допускається.
```
Comment (alinkedd) [inline files/uk/web/html/element/input/time/index.md:147]:
```suggestion
Усталено `<input type="time">` не застосовує до введених значень жодної валідації, окрім того, що інтерфейс користувацького агента загалом не дозволяє ввести нічого, крім значення часу. Це корисно, але не можна повністю покладатися на те, що значення буде коректним часовим рядком, оскільки воно може бути порожнім рядком (`""`), що допускається.
```

File: files/uk/web/html/element/input/time/index.md [hunk 7]
```diff
@@ -200,7 +162,7 @@ _Наразі неясно, що означає значення `any` для `s
 
 {{EmbedLiveSample('zadannia-maksymalnoho-ta-minimalnoho-chasu', 600, 40)}}
 
-Ось CSS, використаний у прикладі вище. Тут використовуються властивості CSS {{cssxref(":valid")}} та {{cssxref(":invalid")}}, щоб оформити поле на основі того, чи є поточне значення дійсним. Довелося помістити піктограми на {{htmlelement("span")}} поряд з введенням, а не на саме поле, оскільки в Chrome згенерований вміст розміщується всередині елемента керування формою, і його не можна ефективно стилізувати або показати.
+Ось CSS, використаний у прикладі вище. Тут використовуються властивості CSS {{cssxref(":valid")}} та {{cssxref(":invalid")}}, щоб оформити поле на основі того, чи є поточне значення дійсним. Піктограма додана як згенерована піктограма content на {{htmlelement("span")}} поряд з полем.
```
Comment (alinkedd) [inline files/uk/web/html/element/input/time/index.md:165]:
```suggestion
Ось CSS, використаний у прикладі вище. Тут використовуються властивості CSS {{cssxref(":valid")}} та {{cssxref(":invalid")}}, щоб оформити поле на основі того, чи є поточне значення дійсним. Піктограма додана через згенерований вміст (поле content) в елементі {{htmlelement("span")}} поряд з полем.
```

**Review by alinkedd — CHANGES_REQUESTED** [review]

**Review by alinkedd — APPROVED** [review]

**Pull request merged**